postgres/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 
 * Updated dependencies. ([#16154](https://github.com/DataDog/integrations-core/pull/16154))
 
+***Added***:
```

Comment:
Not sure why `ddev release changelog new` made the changelog for 15.3.0 even though this change isn't available yet?

Reply:
upgrade ddev to v6 should resolve the issue

Reply:
Fixed